### PR TITLE
Study 0.2 draft

### PR DESCRIPTION
**Pull request by AlasdairGray**
Updating study profile based on discussions with plant community, and doing various bits of housekeeping.

Profile is consistent with schema v11.01 and Study v0.3-DRAFT.

0.3 version of the profile will need to incorporate the changes that result from [issue 472](https://github.com/BioSchemas/specifications/issues/472)

### CODE REVIEW

File: _types/Study/0.3-DRAFT.html [hunk 1]
```diff
@@ -0,0 +1,883 @@
...
+- "devTypes/Study/specification/"
+- "/devTypes/Study/"
+- "/Study"
+- "/types/Study"
+
+previous_version: '0.2-DRAFT-2019_06_21'
+previous_release:
+
+dateModified: 2020-12-15
+description: "Study"
```
Comment (ljgarcia) [inline _types/Study/0.3-DRAFT.html:13]: Could this description be improved?

File: _profiles/Study/0.2-DRAFT.html [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+group: studies
+use_cases_url: /useCases/Study/
+cross_walk_url: https://docs.google.com/spreadsheets/d/1320sC6scXagwB5kJyGObuW5YkWGrZ1JNIYk-rmC3pyE/
+gh_tasks: https://github.com/Bioschemas/bioschemas/labels/type%3A%20Study
+live_deploy: ''
+
+parent_type: Study
+hierarchy:
+- Thing
+- CreativeWork
```
Comment (ljgarcia) [inline _profiles/Study/0.2-DRAFT.html:25]: Being this the Study type profile, should not the hierarchy be Thing-CreativeWork-Study?

**Review by ljgarcia — CHANGES_REQUESTED** [review]
In addition to inline comments:
* Type contributors appears empty
* Group has no leaders nor objectives
* Link to examples does not work
* Tasks and issues is the same for the type and the profile
* There are no use cases

**Comment by AlasdairGray** [comment]
> * Group has no leaders nor objectives

Covered in [issue 258](https://github.com/BioSchemas/specifications/issues/258), while not ideal we should still let the work proceed.

**Comment by AlasdairGray** [comment]
> * Tasks and issues is the same for the type and the profile

This is the same for all specifications.

**Comment by AlasdairGray** [comment]
> * Type contributors appears empty

Contributors for types is auto populated from the group membership, so again should not prevent us from publishing this for now.

**Comment by AlasdairGray** [comment]
> * Link to examples does not work

Link now fixed, and enabled live deploy link (bb49eb13dcd19c07e86973f8c56d35ccbde44503)

File: _types/Study/0.3-DRAFT.html (hunk 1 above)
Comment (AlasdairGray) [reply _types/Study/0.3-DRAFT.html:13]: Already raised in issue 286

File: _types/Study/0.3-DRAFT.html [hunk 3]
```diff
@@ -0,0 +1,883 @@
+---
```
[the remark this answers is not in the document]
Comment (AlasdairGray) [reply _types/Study/0.3-DRAFT.html:1]: Not for me, maybe you have visited one but not the other

File: _data/type_versions.yaml [hunk 4]
```diff
@@ -116,7 +116,7 @@ SequenceRange:
 
 Study:
     name: "Study"
-    latest_publication: "0.2-DRAFT-2019_06_21"
+    latest_publication: "0.3-DRAFT"
```
[the remark this answers is not in the document]
Comment (AlasdairGray) [reply _data/type_versions.yaml:119]: No, dates were only put in when version numbers were not unique. It is a lot easier if our version numbers do not have dates.

File: _data/profile_versions.yaml [hunk 5]
```diff
@@ -170,7 +170,7 @@ SemanticAnnotation:
 
 Study:
     name: "Study"
-    latest_publication: "0.1-DRAFT-2018_11_15"
+    latest_publication: "0.2-DRAFT"
```
[the remark this answers is not in the document]
Comment (AlasdairGray) [reply _data/profile_versions.yaml:173]: No, dates were only added when the version numbers were not unique.

**Comment by AlasdairGray** [comment]
@ljgarcia that's all your issues either resolved or tracked as issues for a future draft revision

**Comment by ljgarcia** [comment]
All issues addressed, thanks.

**Review by ljgarcia — APPROVED** [review]

**Pull request merged**